client/packages/sdk/package.json

```diff
@@ -22,6 +22,7 @@
     "build": "yarn clean:build && tsup",
```

Comment:
```
  "name": "@t3rn/sdk",
  "version": "0.4.4",
 
```

update version of sdk so it will be published

Reply:
good